packages/react-focus-management/src/FocusManagementProvider.tsx

```diff
@@ -55,7 +55,10 @@ export const useFocusManagementProvider = (
       createAbilityHelpers(state.document.defaultView, ahOptions);
   }
   // memoize context value so that it's stable
-  state.contextValue = React.useMemo(() => ({ focusable: ahInstance?.focusable, ahInstance }), [ahInstance]);
+  state.contextValue = React.useMemo(
+    () => ({ focusable: ahInstance?.focusable, keyboardNavigationState: ahInstance?.keyboardNavigation, ahInstance }),
```

Comment:
Do we really need to pass down `keyboardNavigationState`? Can we just reuse `ahInstance` in our hooks?

Reply:
yeah you're right, it's not necessary, other APIs that need intiialization (modalizer, deloser) will probably require more thinking but I can remove in this case